wls-vorfaelleundvorkommnisse-service/src/main/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/rest/ereignis/EreignisWriteDTO.java

```diff
@@ -0,0 +1,8 @@
+package de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.rest.ereignis;
+
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.domain.ereignis.Ereignisart;
+
+public record EreignisWriteDTO(String beschreibung,
+                               java.time.LocalDateTime uhrzeit,
```

Comment:
dont use fully qualified name

Reply:
see commit [f58e2b5](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/f58e2b5b367a44e8f46391ea078a62e4a1bcf65a)